### PR TITLE
♻️ Replace pygtail with own alternative

### PR DESCRIPTION
**Pull request by pajowu**
This PR replaces the pygtail dependency with a smaller version I wrote based on pygtail. This version notably does not include a racecondition that pygtail has, which breaks our logparsing on every logrotate until manual intervention

### CODE REVIEW

**Review by stefanw — APPROVED** [review]
Let’s merge it before Sunday 😉

File: Makefile [hunk 1]
```diff
@@ -18,5 +18,5 @@ messagesde:
 	python manage.py makemessages -l de --ignore public --ignore froide-env --ignore node_modules --ignore htmlcov --add-location file
 
 requirements: pyproject.toml
-	uv pip compile -o requirements.txt pyproject.toml
-	uv pip compile -o requirements-test.txt --extra test pyproject.toml
+	uv pip compile -o requirements.txt pyproject.toml -p 3.10
```
Comment (krmax44) [inline Makefile:21]: Is there a way to put that into `pyproject.toml`?

File: compose-dev.yaml [hunk 2]
```diff
@@ -8,7 +8,7 @@ services:
       POSTGRES_DB: froide
       POSTGRES_PASSWORD: froide
     ports:
-      - 127.0.0.1:5432:5432
+      - 127.0.0.1:5433:5432
```
Comment (krmax44) [inline compose-dev.yaml:11]: Is this change intentional?

Comment (pajowu) [reply compose-dev.yaml]: oops, no

File: Makefile (hunk 1 above)
Comment (pajowu) [reply Makefile]: There is, but it brakes the CI
